serverless/eventing/brokers/serverless-connecting-brokers-sinks.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="serverless-connecting-brokers-sinks"]
+= Connecting brokers to sinks
+:context: serverless-connecting-brokers-sinks
+
+toc::[]
+
+Brokers can be used in combination with triggers to deliver events from an event source to an event sink. Events are sent from an event source to a broker as an HTTP `POST` request. After events have entered the broker, they can be filtered by CloudEvent attributes using triggers, and sent as an HTTP `POST` request to an event sink.
+
```

Comment:
The title of the assembly is Connecting brokers to sinks but all the 'Includes' talk about creating Triggers for which we have a separate section. Also, these modules are being reused from the Creating Triggers section, which IMHO is not needed. 

The main work flow here, I think, is 'Delivering events from an event source to an event sink.' 

@pierDipi @gabriel-rh please correct me if I am wrong, but here's my suggestion:
1. We should pull out the [Event Delivery](https://docs.openshift.com/container-platform/4.13/serverless/eventing/brokers/serverless-event-delivery.html) section from Brokers and create a separate heading for it under Event discovery. 
2. We should add the content in this PR as the first module in the Event delivery section.
3. Rename the module as 'Using brokers and triggers to deliver events'.
4. Add an additional resources section in the module and link to https://docs.openshift.com/container-platform/4.13/serverless/eventing/triggers/create-triggers.html
5. Make this module the second one: https://docs.openshift.com/container-platform/4.13/serverless/eventing/brokers/serverless-event-delivery.html#trigger-event-delivery-config_serverless-event-delivery. to sustain the flow.
6. Keep the other sections as is.

@shreyasiddhartha thank you for your work. Let's wait for Pier to respond before you make the changes. Also, let me know if you would like to discuss this further.

Reply:
@rhuss request your guidance please, especially considering the comment here: https://docs.google.com/spreadsheets/d/1eYAktpCGQaH5CHeGJ3W3RzQ-k2FvrjP_gLWymxTnMMU/edit#gid=428341185

Please correct me if I am wrong, the way I understand it is that event source, sink, broker, channel, subscription etc. are building blocks that can be used to create a customized event delivery master piece. Ideally our docs focus more on the user story and to my mind, event delivery seems to be the key workflow/user story in Eventing. If this is correct, then I suggest we bring the Event Delivery section on the level of Brokers etc.

I would further suggest that, if Shreya is ok taking this up, to move the event discovery section under the Event Sources section (make it an assembly under event sources) as all the modules under it deal with event sources and not general eventing. So ideally we would have the event delivery section after Subscriptions section.

Reply:
@shreyasiddhartha we need to add the exact preview link to the description to help reviewers: https://59364--docspreview.netlify.app/openshift-enterprise/latest/serverless/eventing/brokers/serverless-connecting-brokers-sinks.html

Reply:
For 1.  
> We should pull out the [Event Delivery](https://docs.openshift.com/container-platform/4.13/serverless/eventing/brokers/serverless-event-delivery.html) section from Brokers and create a separate heading for it under Event discovery.

I wouldn't put it under Event discovery since Event discovery is a different than event delivery (and we will probably have more content added to it with the future features that we are working on upstream that are cross cutting all other resources).

> Ideally our docs focus more on the user story and to my mind, event delivery seems to be the key workflow/user story in Eventing. If this is correct, then I suggest we bring the Event Delivery section on the level of Brokers etc.

I agree with this!

> I would further suggest that, if Shreya is ok taking this up, to move the event discovery section under the Event Sources section (make it an assembly under event sources) as all the modules under it deal with event sources and not general eventing. So ideally we would have the event delivery section after Subscriptions section.

I wouldn't do this for now, since new features will have more cross cutting content

Reply:
Thank you, @pierDipi, so for now we:
1. Pull out the Event Delivery section from Brokers and bring the Event Delivery section on the level of Brokers etc. and place it after Subscriptions section. 
2. Follow steps 2-5 as mentioned in my first comment.
3. Let Event Delivery stay as is.